### PR TITLE
Provide tokens for new and fixed warnings

### PR DESCRIPTION
**Pull request by sladyn98**
Fixes https://issues.jenkins-ci.org/browse/JENKINS-57312

### CODE REVIEW

**Comment by sladyn98** [comment]
@uhafner Added a test stub for the `tokenIssueSize`. Would the enum be in the `IssueSizeTokenMacro`
Class.

**Comment by uhafner** [comment]
I think the best place for the enum would be in the `io.jenkins.plugins.analysis.core.util` package. Then I can reuse that part in order to customize the column as well (later on in another PR).

File: src/test/java/io/jenkins/plugins/analysis/core/tokens/IssuesSizeTokenMacroTest.java [hunk 1]
```diff
@@ -67,6 +67,15 @@ void shouldExpandTokenOfTwoActions() {
         assertThat(macro.evaluate(run, null, null)).isEqualTo("0");
     }
 
+    @Test
+    void checkTotalNumberOfIssues() {
+        IssuesSizeTokenMacro macro = new IssuesSizeTokenMacro();
+        /**
```
Comment (uhafner) [inline src/test/java/io/jenkins/plugins/analysis/core/tokens/IssuesSizeTokenMacroTest.java]: Here we would have the setter somewhere like: ` makro.setType(IssueNumberType.NEW)`

**Comment by uhafner** [comment]
Are you planning to continue on this PR? Or should I take over to create the new enum?

**Comment by sladyn98** [comment]
I would definitely like to continue work on this tomorrow. I was just too busy with my community bridge internship. I completely forgot about it. Apologies

**Comment by uhafner** [comment]
Ok, good to see. There are no time constraints from my side. I'm just trying to keep the project clean and checking all the PRs from time to time.

**Comment by sladyn98** [comment]
@uhafner I am sorry I do not think I have the time to complete this PR, could you go ahead and create the enum. Thanks and apologies.

**Comment by uhafner** [comment]
No worries, thanks anyway!

**Pull request closed without merging**